legistar/bills.py

```diff
@@ -316,7 +324,30 @@ def history(self, matter_id):
                               action['MatterHistoryActionBodyName'])),
                          key=lambda action: action['MatterHistoryActionDate'])
 
-        return actions
+        # sometimes there are exact duplicates of actions. while this
+        # is a a data entry problem that ideally the source system
+        # would fix, they ain't always the way the world works.
+        #
+        # so, remove adjacent duplicate items.
+        uniq_actions = []
+
+        previous_key = None
+        for action in actions:
```

Comment:
say there are two unique actions on a date and one is duplicated. are the duplicates guaranteed to be sequential, or could it happen that the sort on the action date above returns something like `(vote, pass, vote)`?

Reply:
yes, i wanted to make the smallest change I could make that would work. in the cases where I've run into duplicate actions they were always sequential.

Reply:
i'm not sure whether your answer means it's possible but you haven't seen it, or it's not possible. if it's ever possible for actions not to be sequential, i'd prefer an approach where we don't rely on order, e.g.,

```python
uniq_actions = {}
for action in actions:
    if not uniq_actions.get(key):
        uniq_actions[key] = action
    else:
        self.warning(...)
return list(uniq_actions.values())
```

if it's not possible, then your implementation is fine!

Reply:
it might be possible, but i don't feel confident enough that a more comprehensive solution, as you suggest, wouldn't introduce other bugs. 

I'd prefer to make the smaller change, and if it doesn't solve the problem, make a bigger change with some real examples